model_analyzer/analyzer.py

```diff
@@ -110,7 +110,8 @@ def profile(self, client, gpus):
             self._server.stop()
 
         # Profile all models concurrently
-        if self._config.run_config_profile_models_concurrently_enable:
+        if self._config.run_config_profile_models_concurrently_enable and len(
```

Comment:
turn this into a function

Reply:
Created `_should_profile_models_concurrently()`